kovri.i2p/_i18n/fr/user_guide.md

```diff
@@ -1,81 +1,79 @@
 
-## Step 2. Configure Kovri
+## Etape 2. Configurer Kovri
 
-For a full list of options:
+Pour la liste de toutes les options possibles:
 
-```bash
+```bashfeature requests
 $ ./kovri --help
 ```
 
-For complete detailed options:
+Pour des précisions détaillées concernant chaque option:
 
-- `kovri.conf` configuration file for router and client
-- `tunnels.conf` configuration file for client/server tunnels
+- `kovri.conf` fichier de configuration concernant le routeur et le client
+- `tunnels.conf` fichier de configuration concernant les tunnels client/serveur
 
-## Step 3. Run Kovri
+## Etape 3. Exécuter Kovri
 ```bash
 $ cd build/ && ./kovri
 ```
 
-- Wait 5 minutes or so to get bootstrapped into the network before attempting to use services
+- Attendez environ 5 minutes pour être connecté au réseau avant de faire quoi que ce soit.
 
-## Step 4. Join us on IRC
-1. Startup your [IRC client](https://en.wikipedia.org/wiki/List_of_IRC_clients)
-2. Setup your client to connect to kovri's IRC port (default 6669). This will connect you to the Irc2P network (I2P's IRC network)
-3. Join `#kovri` and `#kovri-dev`
+## Etape 4. Nous rejoindre sur IRC
+1. Démarrez votre [client IRC](https://en.wikipedia.org/wiki/List_of_IRC_clients)
+2. Réglez votre client pour qu'il se connecte au port IRC de kovri (6669 par défaut). Cela vous connectera au réseau internet Irc2 (le réseau IRC de I2P).
+3. Rejoignez `#kovri` et `#kovri-dev`
 
-## Step 5. Browse an I2P website (garlic-site/eepsite)
-1. Startup a browser of your choosing (preferably a browser devoted to kovri usage)
-2. Configure your browser by reading [these instructions](https://geti2p.net/en/about/browser-config) **but instead of port 4444 and 4445** change HTTP proxy port to **4446** and SSL proxy port *also* to **4446**
-3. Visit http://check.kovri.i2p
+## Etape 5. Ouvrir le site internet de I2P (garlic-site/eepsite)
+1. Ouvrez le navigateur de votre choix (de préférence compatible avec kovri)
+2. Configurez votre navigateur en lisant [ces instructions](https://geti2p.net/en/about/browser-config) **mais à la place du port 4444 et 4445** changez le port proxy HHTP en **4446** et **également** le port proxy SSL en **4446**
+3. Rendez-vous sur http://check.kovri.i2p pour vérifier que cela fonctionne
 
 Notes:
 
-- **Just like with Tor, one doesn't need SSL to safely and securely use the network**
-- SSL site support and outproxy service is not currently implemented
-- If someone gives you a .i2p address that's not in your address book, use the `Jump` service at http://stats.i2p/i2p/lookup.html
-- Look through hosts.txt in your data directory to view a list of default sites you can easily visit
-- Overall, HTTP Proxy and address book implementation are in development and not yet feature-complete
+- **Exactement de la même façon que Tor, SSL n'est pas nécessaire pour utiliser sans risque et de façon sûre notre réseau**
```

Comment:
well, it's not really "our network" :), i would change "notre réseau" with "le réseau"